internal/platform/writer/console.go

```diff
@@ -21,13 +21,35 @@ var _ scan.Writer = Console{}
 // Console is a [scan.Writer] implementation that writes the output
 // to the given [io.Writer], following console/terminal standards
 // in a human-friendly fashion.
+
+type ConsoleOption func(*Console)
```

Comment:
Note that the comments above are the GoDoc for `type Console ...`, so you should move `type ConsoleOption` to either below the `type Console` or above these comment lines.

Reply:
done